src/Core/src/Handlers/Border/BorderHandler.iOS.cs

```diff
@@ -63,13 +63,7 @@ public override void SetVirtualView(IView view)
 
 		public override void PlatformArrange(Rect rect)
```

Comment:
This is part of shipped public API so I didn't want to remove the method entirely unless you say I can do it.

Reply:
If you can, you can remove and then copy the shipped entry to unshipped, but prefix with `*REMOVED*` and then things should be OK.

Probably not a big issue, but popping in and out of dotnet code probably wastes a few ns in perf.

Reply:
Let me do that.

May you expand on the second sentence regarding perf? What are you referring to?

Reply:
@mattleibow I think I've got what you meant, probably referring to CA transaction started even when not needed, so I refactored the code.